crates/ra_ide/src/syntax_highlighting.rs

```diff
@@ -281,3 +291,44 @@ fn highlight_name_by_syntax(name: ast::Name) -> Highlight {
         _ => default,
     }
 }
+
+fn highlight_injection(
+    acc: &mut Vec<HighlightedRange>,
+    sema: &Semantics<RootDatabase>,
+    literal: ast::RawString,
+    expanded: SyntaxToken,
+) -> Option<()> {
+    let call_info = call_info_for_token(&sema, expanded)?;
+    let idx = call_info.active_parameter?;
+    let name = call_info.signature.parameter_names.get(idx)?;
+    if name != "ra_fixture" {
```

Comment:
So to enable the syntax highlight inside string literal, the parameter name should be called "ra_fixture", right ?